specs/grafana_event_handling_in_monitoring_integration.adoc

```diff
@@ -0,0 +1,313 @@
+:imagesdir: ./images
```

Comment:
You branch name should be `spec/grafana_event_handling_in_monitoring_integration`